pkgs/tools/networking/podcast/default.nix

```
@@ -0,0 +1,35 @@
+{ stdenv, fetchpatch, fetchFromGitHub, rustPlatform, pkgconfig, openssl, darwin
+}:
```

```suggestion
{ stdenv, fetchpatch, fetchFromGitHub, rustPlatform, pkg-config, openssl, Security }:
```

Please do not input darwin.